doc/historylog.rst

```diff
@@ -2,26 +2,48 @@
 History
 #######
 
-With the **History** pane, you can view an automatically de-duplicated, time-stamped list of every command you enter into any connected :doc:`ipythonconsole`.
+With the **History** pane, you can view all the commands entered into any :doc:`ipythonconsole` along with their time-stamp.
 
 .. image:: images/history/history-menu.png
    :alt: Spyder History Log, displaying a list of previously executed commands
 
+(Replace screenshot for screenshot without the options menu)
 
 
-=====================
-Using the History Log
-=====================
 
-Navigating the :guilabel:`History Log` is very straightforward.
-Each session is marked by a date and time-stamp, making it easy to remember when you executed a certain command.
+=================
+Using the History
+=================
+
+Navigating the :guilabel:`History` pane is very straightforward.
+Each Spyder session is marked by a date and time-stamp, making it easy to remember when you executed a certain command.
 Statements can be selected and copied from the context menu or with the normal system shortcuts.
 Just like in the editor, highlighting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
-Finally, the top-right options menu (:guilabel:`Gear` icon) allows you to toggle soft-wrapping of long lines (:guilabel:`Wrap lines`), and set the number of commands the :guilabel:`History Log` should remember (:guilabel:`History`).
 
-The :guilabel:`History Log` is stored in the :file:`.spyder-py3` (Python 3) or :file:`spyder` (Python 2) directory in your user home folder (by default, :file:`C:/Users/{username}` on Windows, :file:`/Users/{username}` for macOS, and typically :file:`/home/{username}` on GNU/Linux).
+
+
+============
+Options Menu
+============
+The top-right options menu (:guilabel:`Gear` icon) allows you to set the number of commands the :guilabel:`History` pane should remember.
+
+(Gif with dialog for changing the maximum entries)
+
+You can also toggle wrapping of long lines (:guilabel:`Wrap lines`), and decide whether to display the line number at the left of the pane.
```

Comment:
```suggestion
You can also toggle wrapping of long lines (:guilabel:`Wrap lines`), and whether the line number is displayed to the left of the text (:guilabel:`Show line numbers`).
```

Simplify syntax, avoid ambiguity and be consistent about stating the guilabel name,